hydra-node/src/Hydra/Crypto.hs

```diff
@@ -248,13 +248,14 @@ newtype MultiSignature a = HydraMultiSignature {multiSignature :: [Signature a]}
   deriving newtype (Semigroup, Monoid)
   deriving anyclass (ToJSON, FromJSON)
 
+instance (Arbitrary a, SignableRepresentation a) => Arbitrary (MultiSignature a) where
+  arbitrary = HydraMultiSignature <$> arbitrary
+
```

Comment:
Maybe revert these lines to where they were? I don't see any change a part from location of this arbitrary instance

Reply:
Yes. I only moved it. It is more common to have instances right after the type definition before other module-level functions. Would you say the place was better before?

Reply:
Nah, it is fine as is